src/analysis/retail/warrior/protection/modules/features/Checklist/Component.tsx

```diff
@@ -67,8 +67,15 @@ const ProtectionWarriorChecklist = ({ combatant, castEfficiency, thresholds }: C
           </>
         }
       >
-        <AbilityRequirement spell={SPELLS.SHIELD_WALL.id} />
-        <AbilityRequirement spell={SPELLS.LAST_STAND.id} />
+        {combatant.hasTalent(TALENTS.SHIELD_WALL_TALENT) && (
```

Comment:
These can be simplified to `<TalentCastEfficiencyRequirement spell={TALENT.SHIELD_WALL_TALENT} />`, which will handle the conditional rendering for you.